documentation/book/con-cluster-operator-rbac.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// assembly-operators-cluster-operator.adoc
+
+[id='con-cluster-operator-rbac-{context}']
+= Role-Based Access Control (RBAC)
+
+== Provisioning Role-Based Access Control (RBAC) for the Cluster Operator
+
+For the Cluster Operator to function it needs permission within the {ProductPlatformName} cluster to interact with the resources it manages and interacts with.
+These include the desired resources, such as  `Kafka`, `Kafka Connect`, and so on, as well as the managed resources, such as `ConfigMaps`, `Pods`, `Deployments`, `StatefulSets`, `Services`, and so on.
+Such permission is described in terms of {ProductPlatformName} role-based access control (RBAC) resources:
+
+* `ServiceAccount`,
+* `Role` and `ClusterRole`,
+* `RoleBinding` and `ClusterRoleBinding`.
+
+In addition to running under its own `ServiceAccount` with a `ClusterRoleBinding`, the Cluster Operator manages some RBAC resources because some of the components need access to {ProductPlatformName} resources.
+
+{ProductPlatformName} also includes privilege escalation protections that prevent components operating under one `ServiceAccount` from granting other `ServiceAccounts` privileges that the granting `ServiceAccount` does not have.
```

Comment:
Why is `ServiceAccounts` plural once and singular twice

Reply:
That is an interesting questions. Kubernetes / OpenShift support all resource names with both singular and plural. So you can even inside commands decide whether you use `os get serviceaccounts` or `oc get serviceaccount`. So I guess I usually use it depending whether it should be logically a plural or singular - i.e. whether I'm refering to single `ServiceAccount` or multiple `ServiceAccounts`.